common/src/glob.rs

```diff
@@ -0,0 +1,115 @@
+use std::borrow::Cow;
+
+use globset::GlobBuilder;
+use lazy_static::lazy_static;
+use lru::LruCache;
+use parking_lot::Mutex;
+use regex::bytes::Regex;
+
+lazy_static! {
+    static ref GLOB_CACHE: Mutex<LruCache<(GlobOptions, String), Regex>> =
+        Mutex::new(LruCache::new(500));
```

Comment:
This number is most likely off. I'm pretty sure that with 500 globs, we'll have only evictions in practice, especially if the plan is to use this across all of Sentry.

Reply:
that's the current number :P